src/VisualStudio/Core/Test/ProjectSystemShim/VisualStudioProjectTests/AnalyzerReferenceTests.vb

```diff
@@ -215,24 +202,7 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.ProjectSystemShim
                 project.AddAnalyzerReference(path2)
 
                 AssertEx.Equal({path1, path2}, environment.Workspace.CurrentSolution.Projects.Single().AnalyzerReferences.Select(Function(r) r.FullPath))
-
             End Using
         End Function
-
-        Private Shared Async Function GetDiagnostics(environment As TestEnvironment) As Task(Of ImmutableArray(Of DiagnosticData))
-            ' Wait for diagnostics to be updated asynchronously
-            Dim waiter = environment.ExportProvider.GetExportedValue(Of AsynchronousOperationListenerProvider).GetWaiter(FeatureAttribute.DiagnosticService)
-            Await waiter.ExpeditedWaitAsync()
-
-            Dim diagnosticService = environment.ExportProvider.GetExportedValue(Of IDiagnosticService)
-            Dim diagnostics = Await diagnosticService.GetDiagnosticsAsync(
```

Comment:
this test called through the service to then do work that it validated.  so the service existed just to make this code pass.  it was otherwise not observable.  Removed entirely.